### PR TITLE
adds yourkit integration

### PR DESCRIPTION
**Pull request by ddellacosta**
This adds Yourkit integration into the default wildfly setup.

See also https://github.com/democracyworks/krakenstein/pull/32

### CODE REVIEW

**Review by cap10morgan — CHANGES_REQUESTED** [review]
Can we move all these additions into a separate Dockerfile.yourkit that starts with a `FROM quay.io/democracyworks/wildfly:9.0.2.Final-newrelic` kind of like how we add NewRelic in the `Dockerfile.newrelic` file?

**Comment by ddellacosta** [comment]
@cap10morgan Okay, part of the way there, but this is not right yet--let's check it out together when you get a chance.

File: Dockerfile [hunk 1]
```diff
@@ -25,7 +25,7 @@ RUN sed -ie 's/SSLv3, //' /usr/lib/jvm/java/jre/lib/security/java.security
 
 USER jboss
 
-EXPOSE 9990
+EXPOSE 9990 10001
```
Comment (cap10morgan) [inline Dockerfile]: This port should go into `Dockerfile.yourkit` too.

**Review by cap10morgan — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 1 above)
Comment (ddellacosta) [reply Dockerfile]: Okay, moved.

**Review by cap10morgan — APPROVED** [review]

**Comment by ddellacosta** [comment]
@cap10morgan I've been slow about merging this in, but now I'm wondering if I should even bother, if we are removing wildfly entirely (?)--what do you think?

**Comment by cap10morgan** [comment]
I am indeed removing WildFly as fast as I can. I say this is no longer a priority.

**Comment by cap10morgan** [comment]
@ddellacosta Should we close this one w/o a merge?

**Pull request closed without merging**